.github/workflows/frontend.yml

```diff
@@ -47,22 +47,14 @@ jobs:
           filters: .github/file-filters.yml
           list-files: shell
 
-  typescript-and-lint:
+  typescript:
```

Comment:
will renaming this break anything in getsentry, or some other GHA task?

Reply:
nope, this is encapsulated in the Frontend required check